### PR TITLE
Enhancement: Executed rector PHP 8.2 set

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: rector.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
```
Comment (Nyholm) [inline rector.php:1]: Let's not include this file.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you. 

I appreciate the annotation to attribute changes. 
I think other changes is just noise. They dont help the project in any way. 

Please just remove the rector.php file, then I am happy to merge.

File: src/Subscriber/AutoLabelFromContentSubscriber.php [hunk 2]
```diff
@@ -43,16 +39,16 @@ public function onPullRequest(GitHubEvent $event)
         }
 
         // the PR body usually indicates if this is a Bug, Feature, BC Break or Deprecation
-        if (preg_match('/\|\s*Bug fix\?\s*\|\s*yes\s*/i', $prBody, $matches)) {
+        if (preg_match('/\|\s*Bug fix\?\s*\|\s*yes\s*/i', (string) $prBody, $matches)) {
             $prLabels[] = 'Bug';
         }
-        if (preg_match('/\|\s*New feature\?\s*\|\s*yes\s*/i', $prBody, $matches)) {
+        if (preg_match('/\|\s*New feature\?\s*\|\s*yes\s*/i', (string) $prBody, $matches)) {
```
Comment (stof) [inline src/Subscriber/AutoLabelFromContentSubscriber.php:45]: Those explicit string casts are not needed IMO.

Comment (OskarStark) [reply src/Subscriber/AutoLabelFromContentSubscriber.php:45]: Will check and remove

File: rector.php (hunk 1 above)
Comment (OskarStark) [reply rector.php:1]: removed

File: src/Subscriber/AutoLabelFromContentSubscriber.php (hunk 2 above)
Comment (OskarStark) [reply src/Subscriber/AutoLabelFromContentSubscriber.php]: Done @stof

**Comment by OskarStark** [comment]
Ready to merge from my side

**Comment by OskarStark** [comment]
Fixed psalm in my last commit @Nyholm

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**